code/modules/mob/living/carbon/species.dm

```diff
@@ -1293,13 +1293,25 @@ var/list/has_died_as_golem = list()
 	anatomy_flags = HAS_LIPS | HAS_SWEAT_GLANDS
 
 	default_mutations=list(RAD_IMMUNE)
-	brute_mod = 1.3
```

Comment:
removing brute mod should receive its own line in changelog and note the degree of buff (insectoids now take 100% brute damage, down from 130%)